geschaeftsordnung-vollversammlung.md

```diff
@@ -0,0 +1,82 @@
+# Geschäftsordnung der Vollversammlung der Fachschaft Digital Engineering
+
+Diese Geschäftsordnung wurde am xx.xx.xxxx von der Vollversammlung beschlossen.
+
+## § 1 Geltungsbereich
+
+Die Geschäftsordnung gilt für die Vollversammlung der Fachschaft Digital Engineering an der Universität Potsdam.
+
+## § 2 Tagesordnung
+
+(1) Die Vollversammlung folgt einer Tagesordnung.
+
+(2) Die Tagesordnung muss Teil der fristgerechten Einladung einer Vollversammlung sein.
+
+(3) Bis 14 Tage vor einer Vollversammlung können die einladenden Personen gestellte Anträge in die Tagesordnung aufnehmen.
+
+(4) Bis 7 Tage vor einer Vollversammlung muss die finale Tagesordnung auf dem gleichen Weg wie die Einladung zur Vollversammlung bekanntgegeben werden.
+
+## § 3 Versammlungsleitung
+
+(1) Die Vollversammlung wird von den einladenden Personen geleitet.
+
```

Comment:
ÄA:
(1) Die Vollversammlung wird von dern einladenden Personen eröffnet.

(2) Es erfolgt eine offene Wahl der Sitzungsleitung.

Hintergrund:
Die Versammlungsleitung  hat große Möglichkeiten den Verlauf der Sitzung zu gestalten. Ordnungen werden für kontroverse Situationen geschrieben. In einer solchen sollte die Sitzungsleitung nicht nach "First Einberuf first served" vergeben werden. In nicht-kontroversen Situationen ist das Einsetzen des FSRs (die ja üblicherweise einladen) zur Sitzungsleitung in unter einer Minute mit einfachem Handzeichen gemacht.

Reply:
Finde rein subjektiv und ohne tiefere Begründung die Regelung gar nicht schlecht, dass Einladende = Sitzungsleitung. Vielleicht als möglicher Kompromiss: Einladende = Sitzungsleitung, aber die VV kann jederzeit auf Antrag von -- sagen wir 5 oder 10 Mitgliedern -- eine neue Sitzungsleitung beschließen.?

Reply:
Diese Position wurde im FSR noch einmal bestätigt. Wir planen also keine Übernahme des ÄA. Planst du daher einen ÄA auf der VV zu stellen?